packages/desktop-client/src/components/budget/misc.js

```diff
@@ -396,6 +398,12 @@ function SidebarCategory({
                   onDelete(category.id);
                 } else if (type === 'toggleVisibility') {
                   onSave({ ...category, hidden: !category.hidden });
+                } else if (type === 'apply-single-category-template') {
```

Comment:
Suggestion: what if we add a button/icon beside the `#template` text in the notes instead to run the template? Or add a button at the button of the notes component if it detects a `#template` text?